.stylelintignore

```diff
@@ -1,3 +1,3 @@
-packages/design/foundation.css
 public/foundation.css
```

Comment:
This is a tiny thing, but there is a chance to alphabetize this list while it is small...